test/support/generator/random-value-generator.ts

```diff
@@ -94,6 +94,10 @@ export class RandomValueGenerator {
     return (ACBSID.PARTY_ID.PREFIX + this.stringOfNumericCharacters({ length: lengthExcludingPrefix ?? 6 })) as AcbsPartyId;
   }
 
+  acbsBundleId(lengthExcludingPrefix?: number): AcbsBundleId {
+    return (ACBSID.BUNDLE_ID.PREFIX + this.stringOfNumericCharacters({ length: lengthExcludingPrefix ?? 6 })) as AcbsBundleId;
```

Comment:
Can we please use `concat` function instead.

Reply:
Changed all + to .concat( in this file.